### DISCUSSION TITLE
Mobile wallets not connecting

### DISCUSSION THREAD

**Discussion by balkshamster** [WalletConnect/walletconnect-monorepo#2854] [Developer Support] [answered]
Hi, I am migrating to V2, but the https://github.com/WalletConnect/web3modal-examples/blob/main/walletconnect-modal-auth-html-cdn/main.js example does not work with mobile.

When I scan the QR code with the mobile loopring wallet, I do not see the project logo. When I scan the QR with the gamestop wallet, I get a "No info found".

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/128100383/5b658bd0-a56b-4aa6-a87c-8b56951e8bf0)

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/128100383/306a3945-b1ad-4a5f-8d04-973d4e61acee)

I did find this ticket but it doesn't seem similar to mine:
https://github.com/orgs/WalletConnect/discussions/2802

**Comment by Cali93** [comment, marked as answer]
The example you mention in the issue is using the Auth SDK and currently most of wallets support Sign SDK instead of Auth SDK.
I'd suggest you to refer to [wagmi-html ](https://github.com/WalletConnect/web3modal-examples/tree/main/web3modal-wagmi-html)or [sign-html](https://github.com/WalletConnect/web3modal-examples/tree/main/walletconnect-modal-sign-html) examples.
